windows-driver-docs-pr/image/sample-inf-file-for-a-web-services-scanner.md

```diff
@@ -33,20 +33,14 @@ YourWIADriver.dll=1
 [SourceDisksNames.x86]
 1=%Location%,,,
 
-[SourceDisksFiles.ia64]
-YourWIADriver.dll=1
-
-[SourceDisksNames.ia64]
-1=%Location%,,,
-
 [SourceDisksFiles.amd64]
 YourWIADriver.dll=1
```

Comment:
The SourceDisksFiles section is saying the INF has a file named YourWIADriver.dll but [WSDScanDriver.CopyFiles] is copying a file named MyWIADriver.dll - these two file names should match